### DISCUSSION TITLE
Would have sense to consider `cache` option for `.github/workflows` using `setup-node`?

### DISCUSSION THREAD

**Discussion by oscard0m** [pnpm/pnpm#3860] [Q&A] [answered]
Posting this question because I'm not sure if it would have sense or since it's all about `pnpm` itself maybe is necessary to NOT cache when running CI.

Reference: https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#caching-packages-dependencies

**Comment by zkochan** [comment]
I personally don't use caching with pnpm.

That action is maintained by @KSXGitHub, so maybe he can answer

**Comment by KSXGitHub** [comment, marked as answer]
It is already discussed at https://github.com/pnpm/action-setup/issues/19

We may move our discussion there.
